Expense Tracker/Expense_TrackerApp.swift

```diff
@@ -12,6 +13,15 @@ struct Expense_TrackerApp: App {
         UITabBar.appearance().standardAppearance = tabBarAppearance
         UITabBar.appearance().scrollEdgeAppearance = tabBarAppearance
     }
+    
+    init() {
+        let tokenKey = "userToken"
+        if KeychainWrapper.standard.string(forKey: tokenKey) == nil {
+            let token = "zlJGVnAf4UIiblveTO832O6L"
```

Comment:
nit: секреты лучше не пушить, пусть и пет проект)

На будущее, можно делать так, а кому надо подставят свой для тестирования функционала:

```suggestion
            let token = "YOUR_BEARER_TOKEN"
```